docs/develop/module-specifications/spec-distribution.md

```diff
@@ -16,10 +16,6 @@ Passive distribution means that validators and delegators need to manually colle
 
 Collected rewards are pooled globally and distrubuted to validators and delegators. Each validator has the opportunity to charge delegators commission on the rewards collected on behalf of the delegators. Fees are collected directly into a global reward pool and a validator proposer-reward pool. Due to the nature of passive accounting, whenever changes to parameters which affect the rate of reward distribution occur, withdrawal of rewards must also occur.
 
-### Community pool
```

Comment:
@ericHgorski 

We will still have a community pool in Terra 2.0

Maybe just say this:

"The community pool is a reserve of tokens designated for funding projects that promote further adoption and stimulate growth for the Terra economy."